vignettes/fims-path-maturity.Rmd

```diff
@@ -0,0 +1,614 @@
+---
+title: "FIMS Path: Maturity"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{FIMS Path: Maturity}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+```{css, echo=FALSE}
+
+.rchunk {
+  border: solid #2e75b6;
+}
+.rcppchunk {
+  border: solid #c55a11;
+}
+.fimschunk {
+  border: solid #548235;
+}
+.infochunk {
+  border: solid #262626;
+}
+```
+
+
+<style type="text/css">
+code span.co{
+  color: #8c8c8c; 
+}
+</style>
+
+This vignette describes the hierarchical structure of FIMS by describing the linkages, or *path* through the FIMS model using maturity as an example. The vignette is tailored to FIMS developers or others interested in understanding the FIMS core architecture. The vignette walks through how input from R is carried into the core FIMS calculations. 
```

Comment:
I am not sure what is meant by "input from R"

Reply:
changed to: how user provided input from R is carried into the core FIMS calculations.